README.md

```diff
@@ -18,6 +18,36 @@ end
 
 The docs can be found at [HexDocs](https://hexdocs.pm/membrane_element_sdl).
 
+## Usage
+
+The pipeline below displays a h264 video from a file:
```

Comment:
You may want to point out that it has to be a raw h264 stream (not to be confused with mp4 containing h264 encoded video)